profiles/utils.py

```diff
@@ -173,3 +245,81 @@ def update_full_name(user, name):
     if len(name_parts) > 1:
         user.last_name = ' '.join(name_parts[1:])[:30]
     user.save()
+
+
+def dict_to_style(style_dict):
+    """
+    Transform a dict into a string formatted as an HTML style
+
+    Args:
+        style_dict(dict): A dictionary of style attributes/values
+
+    Returns:
+        str: An HTML style string
+    """
+
+    return '; '.join(['{}: {}'.format(k, v) for k, v in style_dict.items()])
+
+
+def generate_initials(text):
+    """
+    Extract initials from a string
+
+    Args:
+        text(str): The string to extract initials from
+
+    Returns:
+        str: The initials extracted from the string
+    """
+    if not text:
+        return None
+    text = text.strip()
+    if text:
+        split_text = text.split(' ')
+        if len(split_text) > 1:
+            return (split_text[0][0] + split_text[-1][0]).upper()
+        else:
+            return split_text[0][0].upper()
+    return None
+
+
+def generate_svg_avatar(text, size, color, bgcolor):
```

Comment:
Nit: can you rename the `text` param to `name` so we're clearer about what the function is expecting?